pykechain/defaults.py

```diff
@@ -177,5 +184,9 @@
         ['id', 'subject', 'status', 'message', 'team', 'created_at', 'options', 'updated_at'])},
     'expiring_downloads': {'fields': ",".join(
         ['id', 'created_at', 'updated_at', 'expires_at', 'expires_in', 'content'])},
+    'context': {'fields': ",".join(
+        ['id', 'name', 'ref', 'created_at', 'updated_at', 'description', 'tags', 'context_type',
+         'activities', 'options', 'feature_collection', 'start_date', 'due_date']
```

Comment:
Shouldn't we have also the `contexts` specified here?

Reply:
Unsure. I thought the context and contexts field were the same.

Reply:
sharp, corrected it - a plural for the query params _is_ needed.
In the mean time I saw that I forgot to add 'scope' to the list of fields to request. Nice!

Reply:
done